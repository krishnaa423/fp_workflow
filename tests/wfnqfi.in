&CONTROL
outdir='./tmp'
prefix='struct'
pseudo_dir='./ONCVPSP/sg15'
calculation='bands'
tprnfor=.true. 
/

&SYSTEM
ibrav=0
ntyp=1
nat=2
nbnd=14
ecutwfc=20.0
!noncolin=.true.
!lspinorb=.true. 
/

&ELECTRONS
/

&CELL
/

&IONS
/

CELL_PARAMETERS angstrom
   2.7150000000    2.7150000000    0.0000000000
   0.0000000000    2.7150000000    2.7150000000
   2.7150000000    0.0000000000    2.7150000000


ATOMIC_SPECIES
Si 28.085 Si_ONCV_PBE_sr.upf


ATOMIC_POSITIONS angstrom 
Si    0.0000000000    0.0000000000    0.0000000000
Si    1.3575000000    1.3575000000    1.3575000000


K_POINTS crystal
8
   0.0000000000    0.0000000000    0.0010000000    1.0000000000
   0.0000000000    0.0000000000    0.5010000000    1.0000000000
   0.0000000000    0.5000000000    0.0010000000    1.0000000000
   0.0000000000    0.5000000000    0.5010000000    1.0000000000
   0.5000000000    0.0000000000    0.0010000000    1.0000000000
   0.5000000000    0.0000000000    0.5010000000    1.0000000000
   0.5000000000    0.5000000000    0.0010000000    1.0000000000
   0.5000000000    0.5000000000    0.5010000000    1.0000000000

