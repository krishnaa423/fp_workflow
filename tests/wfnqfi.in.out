
     Program PWSCF v.7.2 starts on 17Jul2024 at 21:43:57 

     This program is part of the open-source Quantum ESPRESSO suite
     for quantum simulation of materials; please cite
         "P. Giannozzi et al., J. Phys.:Condens. Matter 21 395502 (2009);
         "P. Giannozzi et al., J. Phys.:Condens. Matter 29 465901 (2017);
         "P. Giannozzi et al., J. Chem. Phys. 152 154105 (2020);
          URL http://www.quantum-espresso.org", 
     in publications or presentations arising from this work. More details at
     http://www.quantum-espresso.org/quote

     Parallel version (MPI), running on     1 processors

     MPI processes distributed on     1 nodes
     29753 MiB available memory on the printing compute node when the environment starts
 
     Waiting for input...
     Reading input from standard input
Warning: card &CELL ignored
Warning: card / ignored
Warning: card &IONS ignored
Warning: card / ignored

     Current dimensions of program PWSCF are:
     Max number of different atomic species (ntypx) = 10
     Max number of k-points (npk) =  40000
     Max angular momentum in pseudopotentials (lmaxx) =  4

     Atomic positions and unit cell read from directory:
     ./tmp/struct.save/
     Atomic positions from file used, from input discarded
 
     Message from routine setup:
     using ibrav=0 with symmetry is DISCOURAGED, use correct ibrav instead
 
     Subspace diagonalization in iterative solution of the eigenvalue problem:
     a serial algorithm will be used

 
     G-vector sticks info
     --------------------
     sticks:   dense  smooth     PW     G-vecs:    dense   smooth      PW
     Sum         295     295     91                 3287     3287     609
 
     Using Slab Decomposition
 


     bravais-lattice index     =            0
     lattice parameter (alat)  =       7.2558  a.u.
     unit-cell volume          =     270.1072 (a.u.)^3
     number of atoms/cell      =            2
     number of atomic types    =            1
     number of electrons       =         8.00
     number of Kohn-Sham states=           14
     kinetic-energy cutoff     =      20.0000  Ry
     charge density cutoff     =      80.0000  Ry
     Exchange-correlation= PBE
                           (   1   4   3   4   0   0   0)

     GPU acceleration is ACTIVE.


     celldm(1)=   7.255773  celldm(2)=   0.000000  celldm(3)=   0.000000
     celldm(4)=   0.000000  celldm(5)=   0.000000  celldm(6)=   0.000000

     crystal axes: (cart. coord. in units of alat)
               a(1) = (   0.707107   0.707107   0.000000 )  
               a(2) = (   0.000000   0.707107   0.707107 )  
               a(3) = (   0.707107   0.000000   0.707107 )  

     reciprocal axes: (cart. coord. in units 2 pi/alat)
               b(1) = (  0.707107  0.707107 -0.707107 )  
               b(2) = ( -0.707107  0.707107  0.707107 )  
               b(3) = (  0.707107 -0.707107  0.707107 )  


     PseudoPot. # 1 for Si read from file:
     ./ONCVPSP/sg15/Si_ONCV_PBE_sr.upf
     MD5 check sum: 8553c464762ac42b85b1eed164b4d029
     Pseudo is Norm-conserving, Zval =  4.0
     Generated using ONCVPSP code by D. R. Hamann
     Using radial grid of 1510 points,  4 beta functions with: 
                l(1) =   0
                l(2) =   0
                l(3) =   1
                l(4) =   1

     atomic species   valence    mass     pseudopotential
        Si             4.00    28.08500     Si( 1.00)

     48 Sym. Ops., with inversion, found (24 have fractional translation)



   Cartesian axes

     site n.     atom                  positions (alat units)
         1           Si  tau(   1) = (   0.0000000   0.0000000   0.0000000  )
         2           Si  tau(   2) = (   0.3535534   0.3535534   0.3535534  )

     number of k points=     8
                       cart. coord. in units 2pi/alat
        k(    1) = (   0.0007071  -0.0007071   0.0007071), wk =   0.2500000
        k(    2) = (   0.3542605  -0.3542605   0.3542605), wk =   0.2500000
        k(    3) = (  -0.3528463   0.3528463   0.3542605), wk =   0.2500000
        k(    4) = (   0.0007071  -0.0007071   0.7078139), wk =   0.2500000
        k(    5) = (   0.3542605   0.3528463  -0.3528463), wk =   0.2500000
        k(    6) = (   0.7078139  -0.0007071   0.0007071), wk =   0.2500000
        k(    7) = (   0.0007071   0.7063997   0.0007071), wk =   0.2500000
        k(    8) = (   0.3542605   0.3528463   0.3542605), wk =   0.2500000

     Dense  grid:     3287 G-vectors     FFT dimensions: (  24,  24,  24)

     Estimated max dynamical RAM per process >       2.14 MB

     The potential is recalculated from file :
     ./tmp/struct.save/charge-density

     Starting wfcs are    8 randomized atomic wfcs +    6 random wfcs

     Band Structure Calculation
     Davidson diagonalization with overlap

     ethr =  1.25E-08,  avg # of iterations = 15.5

     total cpu time spent up to now is        0.8 secs

     End of band structure calculation

          k = 0.0007-0.0007 0.0007 (   411 PWs)   bands (ev):

    -5.4902   6.5503   6.5505   6.5505   8.9903   8.9903   8.9903   9.8896
    14.1930  14.1930  14.4756  17.6547  17.6549  17.6549

          k = 0.3543-0.3543 0.3543 (   410 PWs)   bands (ev):

    -3.1382  -0.5092   5.3114   5.3114   8.0373   9.7667   9.7667  14.1763
    17.1043  17.1043  17.8282  17.8282  17.9431  19.5287

          k =-0.3528 0.3528 0.3543 (   410 PWs)   bands (ev):

    -3.1381  -0.5092   5.3113   5.3113   8.0374   9.7667   9.7668  14.1764
    17.1041  17.1049  17.8276  17.8280  17.9437  19.5286

          k = 0.0007-0.0007 0.7078 (   412 PWs)   bands (ev):

    -1.3300  -1.3145   3.6177   3.6177   7.1426   7.1462  16.6129  16.6129
    17.6260  17.6260  19.0421  19.0613  19.5946  19.6098

          k = 0.3543 0.3528-0.3528 (   410 PWs)   bands (ev):

    -3.1381  -0.5092   5.3113   5.3113   8.0374   9.7667   9.7668  14.1764
    17.1041  17.1049  17.8276  17.8280  17.9437  19.5286

          k = 0.7078-0.0007 0.0007 (   412 PWs)   bands (ev):

    -1.3300  -1.3145   3.6177   3.6177   7.1426   7.1462  16.6129  16.6129
    17.6260  17.6260  19.0421  19.0613  19.5946  19.6098

          k = 0.0007 0.7064 0.0007 (   412 PWs)   bands (ev):

    -1.3300  -1.3145   3.6177   3.6177   7.1426   7.1462  16.6129  16.6129
    17.6260  17.6260  19.0421  19.0613  19.5946  19.6098

          k = 0.3543 0.3528 0.3543 (   410 PWs)   bands (ev):

    -3.1381  -0.5092   5.3113   5.3113   8.0374   9.7667   9.7668  14.1764
    17.1041  17.1049  17.8276  17.8280  17.9437  19.5286

     Writing all to output data dir ./tmp/struct.save/
 
     init_run     :      0.02s CPU      0.06s WALL (       1 calls)
     electrons    :      0.57s CPU      0.64s WALL (       1 calls)

     Called by init_run:
     wfcinit      :      0.00s CPU      0.00s WALL (       1 calls)
                                        0.00s GPU  (       1 calls)
     potinit      :      0.01s CPU      0.04s WALL (       1 calls)
     hinit0       :      0.01s CPU      0.01s WALL (       1 calls)

     Called by electrons:
     c_bands      :      0.57s CPU      0.64s WALL (       1 calls)
     v_of_rho     :      0.01s CPU      0.04s WALL (       1 calls)

     Called by c_bands:
     init_us_2    :      0.00s CPU      0.00s WALL (       8 calls)
     init_us_2:gp :      0.00s CPU      0.00s WALL (       8 calls)
     cegterg      :      0.50s CPU      0.53s WALL (       8 calls)

     Called by *egterg:
     cdiaghg      :      0.18s CPU      0.20s WALL (     132 calls)
                                        0.20s GPU  (     132 calls)
     h_psi        :      0.22s CPU      0.26s WALL (     140 calls)
                                        0.26s GPU  (     140 calls)
     g_psi        :      0.00s CPU      0.00s WALL (     124 calls)
                                        0.00s GPU  (     124 calls)

     Called by h_psi:
     h_psi:calbec :      0.04s CPU      0.10s WALL (     140 calls)
                                        0.09s GPU  (     140 calls)
     vloc_psi     :      0.11s CPU      0.12s WALL (     140 calls)
                                        0.12s GPU  (     140 calls)
     add_vuspsi   :      0.02s CPU      0.03s WALL (     140 calls)
                                        0.03s GPU  (     140 calls)

     General routines
     calbec       :      0.03s CPU      0.05s WALL (     140 calls)
     fft          :      0.00s CPU      0.00s WALL (      10 calls)
                                        0.00s GPU  (      10 calls)
     fftw         :      0.01s CPU      0.05s WALL (     280 calls)
                                        0.04s GPU  (     280 calls)
     davcio       :      0.00s CPU      0.01s WALL (      16 calls)
 
     Parallel routines
 
     PWSCF        :      0.65s CPU      0.89s WALL

 
   This run was terminated on:  21:43:58  17Jul2024            

=------------------------------------------------------------------------------=
   JOB DONE.
=------------------------------------------------------------------------------=
Warning: ieee_invalid is signaling
Warning: ieee_inexact is signaling
FORTRAN STOP
